### ISSUE TITLE
 Pb with videos in Microsoft Learning Pathways 

### ISSUE THREAD

**Issue by Magali12345** [pnp/custom-learning-office-365#767]
Hello, 

I'm having problems with the content on the SharePoint Learning Pathways site. For a few days now, the video content of the training modules is no longer being displayed and is showing an error (see screenshot below). The same problem is occurring on all the sites I've installed for my customers. 
Were these videos hosted on Stream, which would explain why they were not available? 

Thank you in advance for your feedback 

<img width="1039" alt="image" src="https://github.com/pnp/custom-learning-office-365/assets/136699809/4d474fde-2ba0-41ea-83d9-e0c54c7d0b86">

**Comment by smcclements** [comment]
I'm experiencing the same issue with error message on all videos - 
![image](https://github.com/pnp/custom-learning-office-365/assets/57961998/ec3cd0ea-317c-40c2-9a31-86fdaabc5fae)

**Comment by dcashpeterson** [comment]
@Magali12345 and @smcclements Thank you for bringing this to our attention. This looks like an issue with the source content Learning Pathways uses. We are tracking it down and reporting it to the correct folks. I will keep you posted.

**Comment by MatthieuPanisset** [comment]
Hello, I'm experiencing the same issue: 
![image](https://github.com/pnp/custom-learning-office-365/assets/1810683/69b9ea45-8636-439b-bea4-dc1d3853bf31)
thanks in advance,
Matthieu

**Comment by dcashpeterson** [comment]
@MatthieuPanisset as I mentioned this is an outage with the provider of the videos. I will update this issue when the issue is resolved.

**Comment by PiaStider** [comment]
Same problem as others not able to show videos. 
![Skärmbild 2024-01-24 132337](https://github.com/pnp/custom-learning-office-365/assets/157601940/1ffe7a8f-e013-4c11-b506-85727c720aa4)

**Comment by dcashpeterson** [comment]
@clarelcampbell there is no resolution time at this point. The team responsible for the videos has been notified and they are working on it. I will update here when things are resolved.

**Comment by matthew-kidd-SP** [comment]
Having the same issue here, is there any timescale on when this will be resolved? as its causing a delay to our release of the site
![image](https://github.com/pnp/custom-learning-office-365/assets/157823183/c7c2c609-7310-43f9-9dc9-475b28532dc1)

**Comment by dcashpeterson** [comment]
@Matthew-k99 the team is still working on fixing this. I will update the issue when I have something to report.

**Comment by dcashpeterson** [comment]
Hello folks, we have identified a potential fix for the video issue and are in the process of testing it. Once it is validated, we will have a better idea of how long it will take to implement the fix across all the Microsoft Learning Pathways assets.

**Comment by dcashpeterson** [comment]
@clarelcampbell no, this issue is different and a known issue. The content in Learning Pathways is surfaced in an Iframe. Some content is not allowed to be surfaced in an Iframe including some content on learn.microsoft.com. Those pages can't be surfaced in Learning Pathways unfortunatly.

**Comment by clarelcampbell** [comment]
Thanks - wasn't aware of that, really disappointed I can't pull that through. It's creating a manual process. Here is my scenario,  about new teams, as there is no Microsoft supplied content in MLP (learning pathways) so I am pulling through pages about the toggle etc and trying not to recreate. I now have to go in and see if the "known issues with new teams" has updated. If I could show it on MLP this would be avoided. We could put the link on a page outside MLP but this would dilute the message to my 5,000 colleagues that MLP is the place to go and the expected format the content will be in. If you have any contacts in the MLP content roadmap space please pass on - content about new teams adoption needed asap please! "begging emoji"

**Comment by jfunk23** [comment]
@dcashpeterson - did you see the fix gchrisg proposed?  We looked at doing a URL rewrite to fix, but should be simple code update.

**Comment by GoranHusman** [comment]
I changes the site settings "HTML Field Security" to "Allow contributors to insert iframe...", then all videos works for me.

**Comment by dcashpeterson** [comment]
@jfunk23 @gchrisg we have identified the issue and just changing our asset urls to an unsupported endpoint doesn't fix the issue across all 10 languages and all the assets that we have. This isn't an issue with Learning Pathways. It is an issue with the underlying video service endpoint and we are working with that team to resolve the issue. 

Please keep in mind that Learning Pathways is installed in thousands of tenants worldwide and while this is a serious issue we are doing everything we can to resolve it as soon as we can. We can't roll out fixes without adequate testing.

The fix has been identified and validated and is in process of being rolled out worldwide. It should be resolved globally in the next 24-48 hours. Thank you for your patience.

**Comment by clarelcampbell** [comment]
@dcashpeterson will there be any downtime and/or action required please? Thanks

"It should be resolved globally in the next 24-48 hours"

**Comment by dcashpeterson** [comment]
@clarelcampbell there will be no downtime. Although the UI of the pages being served will change.

**Comment by dcashpeterson** [comment]
Hello all, Thank you for your patience. The fix has been rolled out and I can see it in my developer tenant. Two things have changed. 1) The endpoint for the content/videos has changed. 2) The UI has changed for the assets. The background is gray and the header is smaller. We apologize for the inconvenience.

**Comment by MatthieuPanisset** [comment]
> Hello all, Thank you for your patience. The fix has been rolled out and I can see it in my developer tenant. Two things have changed. 1) The endpoint for the content/videos has changed. 2) The UI has changed for the assets. The background is gray and the header is smaller. We apologize for the inconvenience.

Hello [dcashpeterson](https://github.com/dcashpeterson), thanks for the fix. It work's on my tenant customer!

**Comment by GoranHusman** [comment]
Awesome - A big Thank You to all involved. The problem is now solved in my tenant. 🥂🙏

**Comment by Magali12345** [comment]
Same for me, the problem is fixed in my tenant. Thank you all!!

**Comment by renwein** [comment]
I saw today's post and notification about the gray background update, but I can't find any information about why the gray background was added.  This background color actually makes the assets very difficult to read due to the poor contrast, and it is going to cause issues for individuals with vision challenges.  Can the gray background be removed or changed to a _much_ lighter color?

![LP gray background](https://github.com/pnp/custom-learning-office-365/assets/123119748/2b040b3e-8704-41cd-bb9f-113165bfd1ee)
